pkg/networkservice/mechanisms/kernel/hostdevice/server.go

```diff
@@ -0,0 +1,82 @@
+package hostdevice
+
+import (
+	"context"
+
+	"github.com/golang/protobuf/ptypes/empty"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/vishvananda/netns"
+
+	"github.com/networkservicemesh/api/pkg/api/networkservice"
+	"github.com/networkservicemesh/api/pkg/api/networkservice/mechanisms/kernel"
+	"github.com/networkservicemesh/sdk-vppagent/pkg/tools/netnsinode"
```

Comment:
I presume from your comment below that you intend to remove this cross dependency between sdk-vppagent and sdk-kernel in the future?  Having it to get going here is smart, but in the medium term we probably don't want them depending on each other :)